### ISSUE TITLE
Trouble remaping a button on HP Omen laptop 

### ISSUE THREAD

**Issue by Zammy** [randyrants/sharpkeys#468]
I am trying to remap a key that is placed physically where delete button should be. It has "pause" on written on it. Its not media pause as there is a separate key for that. It is the old scroll pause I assume.

When I press it inside SharpKeys it says that the button is (Num Lock (00_45) but then I get a message box saying "You have entered a key that SharpKeys doesn't know about".

Any tips what is going on? I tested other keys and the remaping works but on the only key I want remapped it does not. :( 

Thank you

**Comment by Zammy** [comment]
Ok, after some research this worked: https://www.neox.net/w/2008/02/13/keyboard-remap-pause-break-key-as-del-key/
I am wondering if you should not offer a special feature to remap specifically "pause" to "delete" button?
Also, f the guys in HP who made this stupid decision to put "pause" button where "delete" should have been

**Comment by randyrants** [comment]
No, because Pause is a triple-byte code.  Remapping the first byte of the triple may remap pause/break fine but there's no guarantee that it won't also remap other triple-byte characters.  I cannot justify putting in a bit of code that I know will be a bug for some users and not others.  It's not deterministic enough to make a change.

The good news is that this is GitHub.  Nothing stopping people for doing what they want with the source.

Also, if this is something you're actively in need of, give Microsoft PowerToys or AutoHotKeys a try because they are active remapped (and aren't capped to double-byte values.)
